mama-prd/README.md

```diff
@@ -1,12 +1,14 @@
 # mama-stage-based-reminders
-MAMA Stage Based Reminders Javascript sandbox application for production.
+MAMA Javascript sandbox application for production.
 
 Conversation configuration:
 
 * [Javascript](https://github.com/praekelt/mama-sms/)
 * [Configuration](config.json)
+* [Reports](reports.json)
 
 Raw URLs:
 
 * Javascript (go-mama-sms.js): [Raw](https://raw.githubusercontent.com/praekelt/mama-sms/develop/lib/go-mama-sms.js)
```

Comment:
I'm not quite sure why the production application uses ``go-mama-sms.js`` while the QA one uses ``go-app-default.js``.